lib/percy/cli.rb

```diff
@@ -76,11 +77,14 @@ def run
           options.widths = (options.widths || '').split(',')
 
           raise OptionParser::MissingArgument, 'root folder path is required' if args.empty?
+
           if args.length > 1
             raise OptionParser::MissingArgument, 'only a single root folder path can be given'
           end
+
           root_dir = args.first
 
+          Percy::Cli::Client.new
```

Comment:
Per discussion, move this side-effect or setup inside run_snapshot somehow so that cli.rb only acts as an argument parser.